CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## [3.3.0](https://github.com/IBM/ignition/tree/3.3.0) (2022-02-28)
+[Full Changelog](https://github.com/IBM/ignition/compare/3.2.0...3.3.0)
+
+**Implemented enhancements:**
```

Comment:
Can just remove this section if there are no enhancements

Reply:
Hi @dvaccarosenna , removed the mentioned section.